packages/sdk/client-services/src/packlets/services/service-host.ts

```diff
@@ -201,7 +206,7 @@ export class ClientServicesHost {
     const {
       connectionLog = true,
       transportFactory = createSimplePeerTransportFactory({
-        iceServers: this._config?.get('runtime.services.ice'),
+        iceServers: this._config ? await getIceServers(this._config) : undefined,
```

Comment:
I think the fetch logic should be inside the transport factory. We should send this request as a part of opening a webrtc connection, but cache the response for the subsequent connections.

There's also a case where we might re-request the ice config for some reason (invalidation, network change). No need to handle this now, but the implementation should allow us to cover this case in the future